azext_iot/iothub/params.py

```diff
@@ -138,6 +138,12 @@ def load_iothub_arguments(self, _):
             help="Initial state of twin reported properties for the target device when the simulator is run. "
             "Optional param, only supported for mqtt.",
         )
+        context.argument(
+            "model_id",
+            options_list=["--model-id", "--dtmi"],
+            help="The Digital Twin Model Id the device will report when connecting to the hub.",
```

Comment:
does this model id have to be the same model id of the device? or can it be anything (even made up)? 

wasnt there a property the user could set before? Does that method still work?

Reply:
This model Id gets stored in the twin, the user-settable attribute I believe you are talking about is for d2c messages specifying which component of the pnp interface is sending the message.

Reply:
so a user could have a device with its twin having model A and when simulating the device, input model B for the --model-id argument, correct?